schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+      },
+      "author_link": {
+        "tile": "Author link",
+        "description": "Author-provided link to related resource.",
+        "type": "object",
+        "properties": {
+          "uri": {
+            "type": "string",
+            "format": "uri"
+          },
+          "display_text": {
+            "description": "Descriptive text to associate with the URI.",
+            "type": "string",
+            "minLength": 1
+          },
+          "category": {
+            "title": "Link category",
+            "description": "The type of resource the link resolves to.",
+            "type": "string",
+            "$comment": "TODO: refine this enumeration",
+            "enum": ["related article", "slides", "notes", "software", "dataset", "simulation", "video", "review", "news item", "personal website", "blog entry", "other"]
+          }
+        },
+        "required": [
+          "uri",
+          "category"
+        ]
+      },
+      "unversioned_identifier": {
+        "title": "arXiv canonical identifier",
+        "type": "string",
+        "pattern": "^(([0-9]{4}\\.[0-9]{4,5})|([a-z\\-]+\\/[0-9]{2}[01][0-9]{4}))$"
+      },
+      "versioned_identifier": {
+        "title": "arXiv canonical identifier with version affix",
+        "type": "string",
+        "pattern": "^(([0-9]{4}\\.[0-9]{4,5})|([a-z\\-]+\\/[0-9]{2}[01][0-9]{4}))v[0-9]+$"
+      },
+      "version": {
+
+        "type": "object",
+        "properties": {
+          "content_types": {
+            "title": "List of content types",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/content_type"
+            },
+            "minItems": 1,
+            "uniqueItems": true
+          },
+          "submitter": {
+            "title": "Submitter",
+            "description": "Information about the submitter and possible proxy for the submitter.",
+            "type": "object",
+            "properties": {
+              "submitter": {
+                "$ref": "#/definitions/person"
+              },
+              "proxy_name": {
+                "title": "Proxy name",
+                "description": "Name of the proxy that submitted on the submitter's behalf.",
+                "type": "string",
+                "minLength": 1
+              }
+            },
+            "required": [
+              "submitter"
+            ]
+          },
+          "submitted_date": {
+            "description": "Date this e-print version was submitted.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "publication_date": {
+            "description": "Date this e-print version was announced.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "license": {
+            "title": "License",
+            "description": "The license associated with the content.",
+            "oneOf": [
+              {"$ref": "#/definitions/active_licenses"},
```

Comment:
The ability to do a legacy license is not idea.

Maybe we should have a leagcy-abs.json for use when converting existing abs and a ng-abs-2021.json for future abs?

Reply:
Right. This schema is intended for announced content, so will need to support legacy licenses. A submission schema would not allow anything but active licenses.